lib/jaeger/client/http_sender.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module Jaeger
+  module Client
+    class HttpSender
+      def initialize(host:, port:, endpoint:, headers:, encoder:, logger: Logger.new(STDOUT))
+        @encoder = encoder
+        @logger = logger
+
+        @uri = URI(host)
```

Comment:
instead of host + port + endpoint, could we just use `url:` like we do in zipkin tracer https://github.com/salemove/zipkin-ruby-opentracing ?